tutorial/debezium-db2-init/db2server/setup_db2_instance.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
```

Comment:
@jcechace Is this file really needed? I don't see it used anywhere

Reply:
You are correct. That is the orignal from ibm image